doc/development_guide/run.rst

```diff
@@ -0,0 +1,83 @@
+===========================================
+ Running Pylint from another python program
+===========================================
+
+You can call Pylint from another Python program thanks to the ``Run()``
+function in the ``pylint.lint`` module
+(assuming Pylint options are stored in a list of strings ``pylint_options``) as:
+
+.. sourcecode:: python
+
+  import pylint.lint
+  pylint_opts = ['--disable=line-too-long', 'myfile.py']
+  pylint.lint.Run(pylint_opts)
+
+Another option would be to use the ``run_pylint`` function, which is the same function
+called by the command line. You can either patch ``sys.argv`` or supply arguments yourself:
+
+.. sourcecode:: python
+
+  import pylint
+
+  sys.argv = ["pylint", "your_file"]
+  pylint.run_pylint()
+  # Or:
+  pylint.run_pylint(argv=["your_file"])
+
+To silently run Pylint on a ``module_name.py`` module,
+and get its standard output and error:
+
+.. sourcecode:: python
+
+  from pylint import epylint as lint
```

Comment:
This example should be updated as `epylint` is not the same as `pylint`. We shouldn't suggest them interchangeably.